.github/workflows/ci-arm64.yml

```diff
@@ -0,0 +1,79 @@
+name: Build OpenSSL Arm64
+
+on:
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+    inputs:
+      openssl_build:
+        description: 'OpenSSL Arm64'
+        required: false
+        default: true
+        type: boolean
+      openssl_clang_build:
+        description: 'OpenSSL with clang-cl Arm64'
+        required: false
+        default: true
+        type: boolean
+
+jobs:
+  build-openssl-arm64:
+    runs-on: [self-hosted, Windows, ARM64, GCC]
+    timeout-minutes: 600
+
+    steps:
+    
+    - name: Build OpenSSL Arm64
+      shell: cmd
+      run: |
+        echo on
+        rmdir /s /q openssl
+        git clone https://github.com/openssl/openssl.git --single-branch --depth 1
```

Comment:
Is there any specific reason why the standard [https://github.com/actions/checkout](https://github.com/actions/checkout) is not used as a separate step?

Reply:
Not really, it has been changed